package.json

```diff
@@ -162,7 +162,7 @@
     ]
   },
   "dependencies": {
-    "@gnosis.pm/safe-apps-sdk": "0.3.1",
+    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk.git#7d399e171319e17e26294d713beb01ceda78fb45",
```

Comment:
Should we merge it with this?

Reply:
Do you think we should publish a dev version to npm?

Reply:
Since our code is open source I think installing the lib from git is fine until we properly test it/address feedback from GP team and make a release

Reply:
I want to avoid that this is by any chance released. If we merge it to development this could happen. 

Are there any circular dependencies?

Reply:
Well, other changes in this PR are dependant on the changes in sdk. I'd say let's just be careful and make use of release branches